internal/resource/config/idp/idp_default_urls_resource.go

```diff
@@ -223,7 +229,15 @@ func (r *idpDefaultUrlsResource) Update(ctx context.Context, req resource.Update
 		diags.AddError("There was an issue retrieving the response of the Idp Default Urls: %s", responseErr.Error())
 	}
 	// Read the response
-	readIdpDefaultUrlsResponse(ctx, updateIdpDefaultUrlsResponse, &state, &plan)
+	// Get the current state to see how any attributes are changing
```

Comment:
nit: can probably remove this comment, it looks copied from PD and isn't really what this is doing